### PR TITLE
Testing Your Ruby Code - Tic Tac Toe PR

### PR DESCRIPTION
**Pull request by BigWizzo**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 gem 'colorize'
+gem 'rspec', '~>3.10.0'
 gem 'rubocop', '~>0.81.0'
 source 'https://rubygems.org'
```
Comment (kitpao) [inline Gemfile]: - [x] **OPTIONAL** The convention in the Gemfile is to put the source at the top and the gems from that source just below, you can find [more info here](https://bundler.io/gemfile.html). In this case: 
  ```ruby
  source 'https://rubygems.org'

  gem 'colorize'
  gem 'rspec', '~>3.10.0'
  gem 'rubocop', '~>0.81.0'
  ```

File: spec/examples.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+example_id                        | status  | run_time        |
+--------------------------------- | ------- | --------------- |
+./spec/tic_tac_toe_spec.rb[1:1:1] | unknown |                 |
+./spec/tic_tac_toe_spec.rb[1:1:2] | failed  | 0.00023 seconds |
+./spec/tic_tac_toe_spec.rb[1:1:3] | unknown |                 |
+./spec/tic_tac_toe_spec.rb[1:1:4] | unknown |                 |
+./spec/tic_tac_toe_spec.rb[1:1:5] | unknown |                 |
+./spec/tic_tac_toe_spec.rb[1:1:6] | unknown |                 |
```
Comment (kitpao) [inline spec/examples.txt]: - [x] Remember to not commit files generated from debugging, in this case, your examples.txt file that shows the status of your tests. Please add this file to .gitignore

File: spec/tic_tac_toe_spec.rb [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    end
+
+    it 'validate Free spaces in the board' do
+      expect(game_ui.entry_space_validator(board, 3)).to eq(3)
+    end
+
+    it 'validate Player name' do
+      allow(game_ui).to receive(:gets).and_return('1')
+      expect { game_ui.entry_space_validator(board, 4) }.to output("That spot is taken chose again\n").to_stdout
+    end
```
Comment (kitpao) [inline spec/tic_tac_toe_spec.rb]: - [x] The name of your test doesn't go with the test you are actually doing, please add a related name

File: spec/tic_tac_toe_spec.rb [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    it 'lets me add a title ' do
+      expect(game_ui.display_title_on_screen).to eq(title)
+    end
+
+    it 'validate the X and O' do
+      allow(game_ui).to receive(:gets).and_return('O')
+      expect { game_ui.validating_value(' ') }.to output("Please enter X or O\n").to_stdout
+    end
+
+    it 'validate that the imput is between 1 to 9 or q' do
```
Comment (kitpao) [inline spec/tic_tac_toe_spec.rb]: - [x] **OPTIONAL** You have a typo here "imput", correct one is input

**Review by kitpao — CHANGES_REQUESTED** [review]

**Review by Temmarie — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hi Team! 👋 

### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: :tada:

### You may merge, good luck with your next projects! :rocket:
### Happy Coding! :+1:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Pull request merged**